metadata.rb

```diff
@@ -4,7 +4,7 @@
 license           'Apache 2.0'
 description       'Installs/Configures aet'
 long_description  'Installs/Configures aet'
-version           '4.0.2-SNAPSHOT'
+version           '4.0.3'
```

Comment:
Latest version is 4.0.1, please leave the old version (`4.0.2-SNAPSHOT`). It will be bumped when new version is released.